src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -193,4 +196,4 @@ public void testRunResultAsync() throws SaltStackException {
         assertTrue(retvals.containsKey("minion-1"));
         assertEquals(retvals.get("minion-1"), true);
     }
-}
+}
```

Comment:
@gborp Sorry, we currently don't distribute a profile, please just configure your IDE with the most important formatting rules that can be found in the [README] (https://github.com/SUSE/saltstack-netapi-client-java/blob/master/README.md).